scripts/authors/index.js

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env node
+
+const { promises: { writeFile } } = require('fs')
+const { join } = require('path')
+const client = require('../lib/client')
+const sortBy = require('./sortBy')
+
+start()
+
+async function start () {
+  const response = await client({
+    url: 'https://api.github.com/repos/omrilotan/isbot/contributors',
+    headers: {
+      'Content-Type': 'application/json',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/omrilotan/isbot/pullRequest?prid=7728710)
```suggestion
      "Content-Type": 'application/json',
```